evdev/src/Evdev/LowLevel.chs

```diff
@@ -140,6 +141,20 @@ withAbsInfo AbsInfo{..} f = do
     pf <- newForeignPtr_ p
     withForeignPtr pf f
 
+--TODO can c2hs make this simpler at all?
+foreign import ccall safe "Evdev/LowLevel.chs.h libevdev_get_abs_info"
+  libevdev_get_abs_info :: Ptr Device -> CUInt -> IO (Ptr ())
+getAbsInfo :: Device -> Word32 -> IO (Maybe AbsInfo)
+getAbsInfo dev x = withDevice dev $ handleNull (pure Nothing) \devPtr -> do
```

Comment:
`handleNull` should be used on `absinfoPtr` not `devPtr`?

Reply:
Thanks 🤦